src/services/address_service.js

```diff
@@ -65,6 +65,17 @@ export default (easypostClient) =>
       return this._all(url, params);
     }
 
+    /**
+     * Retrieve the next page of specific collection of object
```

Comment:
nit: I'd prefer we word all of these something like this:

```suggestion
     * Retrieve the next page of a collection of objects
```